tools/README.md

```diff
@@ -26,16 +26,30 @@ git clone --recurse-submodules https://github.com/commaai/openpilot.git
 
 ``` bash
 cd openpilot
-git lfs pull
 tools/ubuntu_setup.sh
 ```
 
+**3. Git LFS**
+
+``` bash
+git lfs pull
+```
+
+**4. Reset your shell**
+
+Open a new shell or configure your active shell by running:
+``` bash
+source ~/.bashrc
+```
+
+**5. Activate a python shell**
+
 Activate a shell with the Python dependencies installed:
 ``` bash
-poetry shell
+openpilot_shell
```

Comment:
`source .env/bin/activate` isn't super nice, but it is standard and familiar to anyone who works with python projects. we should only add new commands and special knowledge when needed

Reply:
agree. We will still have to deal with the .env since we don't have the poetry plugin anymore

Reply:
oh i don't think we need to replace that, though we can find a good tool for automatic venv activation and recommend it